docs/configuration.rst

```diff
@@ -35,28 +35,41 @@ Variable Substitution
 
 There are following environment variables available in ``molecule.yml``:
 
-================================  ===========================================================
-  Variable                          Description
-================================  ===========================================================
-MOLECULE_DEBUG                    If debug is turned on or off
-MOLECULE_FILE                     Path to molecule config file
-MOLECULE_ENV_FILE                 Path to molecule environment file
-MOLECULE_STATE_FILE               ?
-MOLECULE_INVENTORY_FILE           Path to generated inventory file
-MOLECULE_EPHEMERAL_DIRECTORY      Path to generated directory, usually
-                                  ``~/.cache/molecule/<scenario-name>``
-MOLECULE_SCENARIO_DIRECTORY       Path to scenario directory
-MOLECULE_PROJECT_DIRECTORY        Path to your project directory
-MOLECULE_INSTANCE_CONFIG          ?
-MOLECULE_DEPENDENCY_NAME          Dependency type name, usually 'galaxy'
-MOLECULE_DRIVER_NAME              Name of the molecule scenario driver
-MOLECULE_PROVISIONER_NAME         Name of the provisioner tool (usually 'ansible')
-MOLECULE_REPORT                   Name of HTML file where to dump execution report.
-MOLECULE_SCENARIO_NAME            Name of the scenario
-MOLECULE_VERBOSITY                Determine Ansible verbosity level.
-MOLECULE_VERIFIER_NAME            Name of the verifier tool (usually 'ansible')
-MOLECULE_VERIFIER_TEST_DIRECTORY  ?
-================================  ===========================================================
+MOLECULE_DEBUG
```

Comment:
why reformat it like this?

Reply:
IIRC, rST docs suggest using definition lists for documenting variables. So this change does make sense. @ssbarnea Did doc8 suggest that change?

Reply:
Not really but when I reached the very long lines of that table, I found myself forced to fix that table bug.

doc8 is not as smart to do that.